py/dynruntime.mk

```diff
@@ -29,9 +29,10 @@ CFLAGS += -Wall -Werror -DNDEBUG
 CFLAGS += -DNO_QSTR
 CFLAGS += -DMICROPY_ENABLE_DYNRUNTIME
 CFLAGS += -DMP_CONFIGFILE='<$(CONFIG_H)>'
-CFLAGS += -fpic -fno-common
-CFLAGS += -U _FORTIFY_SOURCE # prevent use of __*_chk libc functions
-#CFLAGS += -fdata-sections -ffunction-sections
+
+MICROPY_ARCH_CFLAGS += -fpic -fno-common
+MICROPY_ARCH_CFLAGS += -U_FORTIFY_SOURCE # prevent use of __*_chk libc functions
```

Comment:
I'm unsure why these options need to go under a different variable name?  And if they do, how about calling it `CFLAGS_ARCH`?